packages/key-management/src/types.ts

```diff
@@ -38,10 +38,26 @@ export enum KeyRole {
   DRep = 3
 }
 
+export enum MultiSigKeyRole {
```

Comment:
We dont need this, you can use the same key role we already have defined as it can be the same

Reply:
Agree

Reply:
Is there a way to enforce that a wrong KeyRole is not assigned during account key derivation? Because I see the possibility of assigning `Internal` or `DRep`

Reply:
I don't think we need to prevent it on this (low) level. Just like there's nothing technically preventing you from using stake role for payment keys. Applications (Lace) is not deriving the keys directly. `deriveAddress` is called by address discovery and tx builder, which can have unit tests that assert derivation props.

Reply:
Noted. Thank you very much.

Reply:
Fixed on fac71bd